web/components/data/FcViewDataAggregate.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="fc-view-data-aggregate">
+    <h1 class="display-4 pa-5">TODO: implement aggregate view</h1>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'FcViewDataAggregate',
+  props: {
+    locations: Array,
+  },
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  async created() {
+    this.loading = true;
```

Comment:
can we at least do the same `syncLocations()` refactor that we did for `FcViewDataMultiEdit`?